ios/RCTPSPDFKit/RCTPSPDFKitView.m

```diff
@@ -104,6 +104,16 @@ - (UIViewController *)pspdf_parentViewController {
   return nil;
 }
 
+- (void)enterAnnotationCreationMode {
+  [self.pdfController setViewMode:PSPDFViewModeDocument animated:YES];
+  [self.pdfController.annotationToolbarController updateHostView:nil container:nil viewController:self.pdfController];
+  [self.pdfController.annotationToolbarController showToolbarAnimated:YES];
+}
+
+- (void)exitCurrentlyActiveMode {
+  [self.pdfController.annotationToolbarController hideToolbarAnimated:YES];
```

Comment:
I think we should also make sure to hide the document editor, if it was visible, since this method is called `exitCurrentlyActiveMode` and the documentation says `Exits the currently active mode, hiding all toolbars.`.

@irgendeinich Would this be the same behavior we have on Android then?

Reply:
> I think we should also make sure to hide the document editor, if it was visible, since this method is called exitCurrentlyActiveMode and the documentation says Exits the currently active mode, hiding all toolbars..

It already does. I've updated the animation in the main comment to show this behavior.